src/index.js

```diff
@@ -1,35 +1,60 @@
 import './index.css';
```

Comment:
- [ ] Please, try to modify your README file and make it more professional by clearing none using sections and personalizing where needed.